src/components/Agent/DetailsAgentPopover.vue

```diff
@@ -39,8 +39,7 @@ export default {
     }
   },
   methods: {
-    ...mapMutations('target', ['removeTargetEntityToDelete']),
-    ...mapMutations('general', ['addEntityToDelete']),
+    ...mapMutations('general', ['addEntityToDelete', 'removeTargetEntityToDelete']),
```

Comment:
why do we have on Agent component to use this target method removeTargetEntityToDelete ??